index.js

```diff
@@ -13,124 +13,137 @@ let config = require('./config.json');
 let prefix = config.commandPrefix;
 //bot
 client.on('guildMemberAdd', member => {
-	console.log(`${member} Has join ${member.guild.name} server.`);
+  console.log(`${member} Has join ${member.guild.name} server.`);
 });
 
 client.on('ready', () => {
-	console.log('Im ready!');
-	var replServer = repl.start({
-		prompt: 'YuriBot > ',
-	});
-	replServer.context.client = client;
-	invite();
+  console.log('Im ready!');
+  var replServer = repl.start({
+    prompt: 'YuriBot > ',
+  });
+  replServer.context.client = client;
 });
 
 try {
-	client.login(config.token);
+  client.login(config.token);
 } catch (e) {
-	console.log(e.stack)
+  console.log(e.stack)
 }
 
-let detectword = ['กระตุก', 'แลค', 'จอดำ', 'ออนไลน์', 'ค้าง', 'DDwrapper'];
-let DDwrapper = ['กระตุก', 'แลค', 'ค้าง', 'DDwrapper', 'เมนู']
-let yurinet = ['yurinet', 'ออนไลน์', 'online']
-let game = ['game', 'ตัวเกม', 'เกม', 'yuri']
-let black_screen = ['จอดำ', 'black_screen']
 client.on('message', async message => {
```

Comment:
**client.on** must followed after **client.login** that inside **try..catch** block

Reply:
so put that try catch block on bottom line?

Reply:
No, Just ignore this comment. :P